### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by samuelmoiwa**
## Solution description
**CSS:** design contents have been added to match the Figma design.
**HTML:** text content and images has been added to the HTML file to match the Figma design
**Form Spree:** add form space link to contact me form, for data submission.
**Linters files:** linters files configured

## Address affected and ensured
* HTML file
* CSS file
* Linters Config
---

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+
+.contact-me-section {
+  width: 100%;
+  margin-top: 150px;
+  padding-bottom: 75px;
+  background-image: url(images/footer-background.svg);
+}
+
+.contact-me-header {
+  padding-top: 105px;
```
Comment (mgmediaweb) [inline style.css]: I kindly recommend you to use flexbox to place element at the center of the container (do not use margin-top), with this your website will be more responsive to any device size

File: style.css [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+  cursor: pointer;
+}
+
+.introduction-heater {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 300px;
```
Comment (mgmediaweb) [inline style.css]: I kindly recommend you to use the flexbox functions to place elements at the center of the container (do not use margin-top), with this your website will be more responsive to any device size

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi  @samuelmoiwa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost1

## Highlights 💯 
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- Good design applied. ✔️

## Required Changes ♻️

*** Kindly recommend you create a media query section in your CSS file to differentiate the mobile and desktop view, your site will adapt to each device

*** Kindly recommend you remove excessive blank lines in your HTML file, it will make your site look more professional.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,603 @@
...
+  margin-left: 24px;
+  cursor: pointer;
+}
+
+/* ############################################## */
+.list-of-portoflio {
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: repeat(4, auto);
+  width: 100%;
```
Comment (koachcode) [inline style.css]: - Please just as the project requirement stated, you are to use media query in order to get a responsive design. Use **media queries in your CSS  file to make your desktop version look like the template you selected (Desktop: 768px or wider).** This should enable your design to look good on both mobile and Desktop view

##### Your design
image



##### FIgma
image

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @samuelmoiwa , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- No linters :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (samuelmoiwa) [reply style.css]: hello, I have made the changes, but it has been submitted twice already, I only have one review now. so please cross-check before I do the submission. thanks

File: style.css [hunk 4]
```diff
@@ -0,0 +1,1253 @@
...
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  color: #6070ff;
+  background: #fff;
+  border-radius: 8px;
+  border-style: none;
+  font-family: var(--font-family);
+  margin-top: 20px;
+  margin-left: 0;
```
Comment (RolandM99) [inline style.css:698]: - [ ] [**OPTIONAL**] To make sure that your style fit exactly the original design from Figma, I'll suggest you to implement your button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template. check the screenshot below:
image

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @samuelmoiwa, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**